Description: Try to send unpair command before remote or host is down.
Author: Corcodel Marian <corcodel.marian@gmail.com>
Bug-Debian: https://bugs.debian.org/923270

---
Last-Update: 2019-08-18

--- bluez-5.43.orig/src/device.c
+++ bluez-5.43/src/device.c
@@ -77,7 +77,7 @@
 
 #define IO_CAPABILITY_NOINPUTNOOUTPUT	0x03
 
-#define DISCONNECT_TIMER	2
+#define DISCONNECT_TIMER	0
 #define DISCOVERY_TIMER		1
 #define INVALID_FLAGS		0xff
 
